dashboard/person.py

```diff
@@ -0,0 +1,46 @@
+import http.client
+import json
+import urllib
+
+import config
+
+
+class API(object):
+    """Retrieve data from person api as needed.  Will eventually replace Mozillians API"""
+    def __init__(self):
+        """
+        :param session: the flask session to update with userinfo
+        """
+        self.config = config.OIDCConfig()
+        self.person_api_url = 'uhbz4h3wa8.execute-api.us-west-2.amazonaws.com'
```

Comment:
we needa nice url for this soon :)

Reply:
I'll move this out into configuration soon as well.